template/dict_to_native_cpp.njk

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2017 The Bacardi Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "{{idl_dir_name}}/{{name | snakecase}}_convert.h"
+
+#include "core/js_type_traits.h"
+#include "core/native_type_traits.h"
+
+void {{name}}ToNative({{name}}& native, Napi::Env env, Napi::Object object) {
```

Comment:
Is there a reason to have to separate this file?

Reply:
first time, i thought it is hard to get keys of dictionary in interface_cpp.njk.
but when i see codes again, it can be to get keys  in interface_cpp.njk.
so i removed this files.